File: src/main/java/problems/Uri1866_Conta.java
package problems;

import java.io.IOException;
import java.util.Scanner;

public class Uri1866_Conta {
    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);
        int n = ler.nextInt();

        for (int b = 1; b <= n; b++) {
            int j = ler.nextInt();

            if (j % 2 == 0) {
                System.out.println("0");
            } else {
                System.out.println("1");
            }
        }
    }
}

File: src/main/kotlin/problems/Uri1866_Conta.kt
package problems

import java.util.Scanner

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)
    val testes: Int = ler.nextInt()
    for (b in 1..testes) {
        val termosSoma: Int = ler.nextInt()
        if (termosSoma % 2 == 0)
            println("0")
        else
            println("1")
    }
}

/*
URI Online Judge | 1866
Conta
Por Ricardo Martins, IFSULDEMINAS BR Brazil

Timelimit: 1
Dois amigos pedem ao atendente de uma lanchonete propor um desafio, de modo que quem acertasse mais, não precisaria pagar a conta. Então foi proposto o seguinte: Dado o seguinte somatório abaixo, informar o resultado, com uma quantidade de termos no mesmo:

S = 1 - 1 + 1 - 1 + 1 - 1 + 1 - 1 ...

Escreva um programa que, dada uma quantidade de termos, informar o resultado do somatório acima.

Entrada
Um número inteiro C será informado, que será a quantidade de casos de teste. Cada caso de teste inicia com um número inteiro N (1 ≤ N ≤ 1000), indicando a quantidade de termos da soma.

Saída
Para cada caso de teste imprima um número S, que é o resultado da soma dos N termos da expressão.

Exemplo de Entrada	Exemplo de Saída
3

11

7

18

1

1

0

V Olimpíada Interna de Programação do IFSULDEMINAS - OLIP 2015
 */